### DISCUSSION TITLE
如何用tauri来打包shion，用于mac端跨平台使用

### DISCUSSION THREAD

**Discussion by SunQiangLoop** [shion-app/shion#9] [General]
mac上怎么使用shion，可以说一下步骤吗

**Comment by hanaTsuk1** [comment]
现在不支持mac版本

**Reply by Emt-lin** [reply]
@hanaTsuk1 什么时候会支持了，tauri 框架兼容 mac 应该不会有太多工作要做吧（当然只是我的猜测）

**Comment by hanaTsuk1** [comment]
@Emt-lin 
一直有人问 那我就详细说明一下为什么没有mac版本
核心功能**自动监听**依赖于系统api
包含功能（[监听程序切换](https://github.com/shion-app/watcher) 和 [监听音频状态](https://github.com/hanaTsuk1/nodio)）
我没mac 所以写不了mac版本的代码

**没有明确支持mac的日期**
